Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -67,7 +68,6 @@ class ValidBlockBitSet final
 	};
 	std::unique_ptr<u32[]> m_valid_block;
```

Comment:
Should also specify somewhere that this data is being read from the JIT, so changing the type requires modifying the dcbx implementation.